nafas/params.py

```diff
@@ -42,6 +42,9 @@
 
 PROGRAM_END_MESSAGE = "Well done!"
 
+SURVEY_LINK_FORMAT = "https://docs.google.com/forms/d/e/1FAIpQLSdTckweXGhaxtbO7jkSEsYhzyoc2LBebyCdKwQGiTRERt742Q/viewform?usp=pp_url&entry.837690934={field_value}"
```

Comment:
Is it considered good practice to gather all parameters in one string? I'm not sure about this!

Reply:
I think otherwise the survey form would look like much longer which I don't like it to be.